src/BlockInfo.h

```diff
@@ -28,7 +28,7 @@ class cBlockInfo
 	/** How much light do the blocks consume? */
 	NIBBLETYPE m_SpreadLightFalloff;
 
-	/** Is a block completely transparent? (light doesn't get decreased(?)) */
+	/** Is a block completely transparent to blocklight? (light doesn't get decreased(?)) */
```

Comment:
Aren't slabs and stairs considered transparent but stop blocklight spreading?

Reply:
"light doesn't get decreased" is misleading anyways - block light always decreases by 1; that comment would only make sense in regards to skylight.

Reply:
@peterbell10 Yes, slabs and stairs are considered transparent yet stop all light spreading. In fact this PR adds that functionality. 

In truth, the "transparent" property isn't really anything to do with lighting, just that most "transparent" blocks let BlockLight (and often SkyLight) pass through. I will change the comment.